packages/aws/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "2.16.0"
+  changes:
+    - description: Add TargetResponseTime metric to ELB Application metrics.
+      type: enhancement
+      link: https://github.com/elastic/integrations/pull/12345
```

Add the correct PR id.

```suggestion
      link: https://github.com/elastic/integrations/pull/10181
```